recipes/docker-image-guide.md

```diff
@@ -56,7 +56,9 @@ WORKDIR /usr/src/app
 ENV NODE_ENV production
 COPY package*.json .
 # https://cheatsheetseries.owasp.org/cheatsheets/NodeJS_Docker_Cheat_Sheet.html#2-install-only-production-dependencies-in-the-nodejs-docker-image
-RUN npm ci --only=production
+# when NODE_ENV is set to production, npm ci automatically omits dev dependencies
+# https://docs.npmjs.com/cli/v10/commands/npm-ci#omit
+RUN npm ci
```

Comment:
Here is my two cents. It is always better to be explicit and state your intentions. Mainly for two reasons, clarity and future-proofing against unintended behavior.

Reply:
So, put `--omit=dev` flag with `npm ci`?

Reply:
@mbareta yes